mobile/src/main/java/ch/epfl/sdp/mobile/ui/play/PlayScreen.kt

```diff
@@ -24,8 +28,22 @@ import ch.epfl.sdp.mobile.ui.PawniesIcons
 fun PlayScreen(
     state: PlayScreenState,
     modifier: Modifier = Modifier,
-    contentPadding: PaddingValues = PaddingValues(),
+    contentPadding: PaddingValues = PaddingValues()
 ) {
+  val strings = LocalLocalizedStrings.current
+  Column(modifier) {
+    Text(
+        text = strings.playOnlineGames,
+        color = MaterialTheme.colors.primary,
+        style = MaterialTheme.typography.h4,
+        modifier = Modifier.padding(horizontal = 16.dp, vertical = 32.dp))
+    LazyColumn(
+        verticalArrangement = Arrangement.Top,
+        horizontalAlignment = Alignment.CenterHorizontally,
+        modifier = Modifier,
+    ) { items(state.matches) { match -> Match(match, state.onGameItemClick) } }
```

Comment:
Maybe these matches would benefit to be keyed, using a pattern similar to this : https://github.com/epfl-SDP/android/blob/main/mobile/src/main/java/ch/epfl/sdp/mobile/ui/social/SocialScreen.kt#L44.